app/interfaces/api/entities/case_stage.rb

```diff
@@ -0,0 +1,11 @@
+module API
+  module Entities
+    class CaseStage < BaseEntity
+      expose :id
```

Comment:
we should not really be exposing `id` - vendors would then use it and rely on it. we should be enabling search by `unique_code` (which we can guarantee will not change)

Reply:
👍 Have removed id from the case stage endpoint